systems/registry/member/Makefile

```diff
@@ -0,0 +1,45 @@
+include ../../config.mk
+
+.PHONY: integration.test test
+
+build: integration.build
+	@echo Building version: \"$(BIN_VER)\"
+	env CGO_ENABLED=0 GOOS=linux GOARCH=amd64  go build -ldflags='-X github.com/ukama/ukama/systems/registry/member/cmd/version.Version=$(BIN_VER) -extldflags=-static' -o bin/member cmd/server/main.go
+
+test:
+	go test -v ./...
+
+# Go lint
+lint:
+	golangci-lint run
+
+deps:
+	go get google.golang.org/protobuf/cmd/protoc-gen-go \
+			google.golang.org/grpc/cmd/protoc-gen-go-grpc \
+			github.com/ysugimoto/grpc-graphql-gateway/protoc-gen-graphql/...
+			github.com/vektra/mockery/v2/.../
+
+
+gen:
+	protoc -I ./pb -I ../../common/pb -I ../../common/pb/dep --go_out=$(GOPATH)/src --go-grpc_out=$(GOPATH)/src --go-grpc_opt=require_unimplemented_servers=true --govalidators_out=$(GOPATH)/src \
```

Comment:
We need to fix the proto gen path so that generated files are place in the the right pb/ sub-folder  instead of created a new github.com/... tree.